pkg/loadbalancer/ipvs.go

```diff
@@ -88,17 +100,29 @@ func NewIPVSLB(address string, port int, forwardingMethod string) (*IPVSLoadBala
 		log.Warnf("unknown forwarding method. Defaulting to Local")
 	}
 
+	if backendHealthCheckInterval <= 0 {
+		backendHealthCheckInterval = 5
+	}
+
 	lb := &IPVSLoadBalancer{
 		Port:                port,
 		client:              c,
 		loadBalancerService: svc,
 		forwardingMethod:    m,
+		interval:            backendHealthCheckInterval,
+		backendMap:          make(map[Backend]bool),
 	}
+
+	if strings.ToLower(forwardingMethod) == "masquerade" {
+		go lb.healthCheck()
+	}
+
 	// Return our created load-balancer
 	return lb, nil
 }
 
 func (lb *IPVSLoadBalancer) RemoveIPVSLB() error {
+	close(lb.stop)
```

Comment:
Hi @lou-lan just curious, has the deletion on iptabales, ipvs and VIP ever worked in your setup if you delete kube-vip?